packages/apptive_grid_core/test/form_data_test.dart

```diff
@@ -323,6 +355,10 @@ void main() {
     test('a != c', () {
       expect(a, isNot(c));
     });
+
+    test('properties hash', () {
+      expect(a.properties.hashCode, b.properties.hashCode);
+    });
```

Comment:
I rather meant that you'd add tests for equality, hashcode and the toString method for the `FormDataProperties` as a new testy group for the properties.

For reference here is testcode for the `CurrencyDataEntity` from `currency_test.dart`

```
  group('DataEntity', () {
    test('Equality', () {
      final a = CurrencyDataEntity(currency: 'USD', value: 12345.12);
      final b = CurrencyDataEntity(currency: 'USD', value: 12345.12);
      final c = CurrencyDataEntity(currency: 'EUR', value: 12345.12);
      expect(a, equals(b));
      expect(a, isNot(c));
    });

    test('Hashcode', () {
      final currency = CurrencyDataEntity(currency: 'USD', value: 12345.12);

      expect(currency.hashCode, Object.hash(12345.12, 'USD'));
    });

    test('toString()', () {
      final currency = CurrencyDataEntity(currency: 'USD', value: 12345.12);

      expect(currency.toString(), equals('CurrencyDataEntity(12345.12 USD)'));
    });
  });
```